AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -821,7 +830,7 @@ - (void)transitionLayoutWithAnimation:(BOOL)animated
     return;
   }
   
-  [self invalidateCalculatedLayout];
+  [self setNeedsLayout];
```

Comment:
I think it is very impressive and awesome we don't have to call invalidateSize here, because you were able to build that in to happen automatically!!

If the new layout requires a different size for ourself, e.g. we are growing as part of the transition adding a new item to our node's stack layout, that gets detected in the layout pass :)